content/sensu-go/6.7/observability-pipeline/observe-schedule/checks.md

```diff
@@ -1431,104 +1793,126 @@ splay_coverage: 90
+**NOTE**: If provided, you must also provide the value for the same metric point tag.
+{{% /notice %}}
+required     | false
 type         | String
 example      | {{< language-toggle >}}
 {{< code yml >}}
-name: instance
+name: namespace
 {{< /code >}}
 {{< code json >}}
 {
-  "name": "instance"
+  "name": "namespace"
 }
 {{< /code >}}
 {{< /language-toggle >}}
 
 value        | 
 -------------|------
-description  | Value for the [output metric tag][19]. Use [check token substitution][39] syntax for the `value` attribute, with dot-notation access to any event attribute.
-required     | true
+description  | Tag value of the metric to use for [metric threshold evaluation][79]. If provided, must match the [event.metrics.points[].tags.value][81] value for a metric point in the check results.{{% notice note %}}
+**NOTE**: If provided, you must also provide the name for the same metric point tag.
+{{% /notice %}}
+required     | false
 type         | String
 example      | {{< language-toggle >}}
 {{< code yml >}}
-value: {{ .name }}
+value: production
 {{< /code >}}
 {{< code json >}}
 {
-  "value": "{{ .name }}"
+  "value": "production"
 }
 {{< /code >}}
 {{< /language-toggle >}}
 
-#### `secrets` attributes
+#### Thresholds attributes
 
-name         | 
+max          | 
 -------------|------
-description  | Name of the [secret][56] defined in the executable command. Becomes the environment variable presented to the check. Read [Use secrets management in Sensu][59] for more information.
-required     | true
+description  | Maximum threshold for the metric for [metric threshold evaluation][79]. You must provide a thresholds `max` value if you do not provide a `min` value.
+required     | false (if a thresholds `min` value is provided)
 type         | String
 example      | {{< language-toggle >}}
 {{< code yml >}}
-name: ANSIBLE_HOST
+max: '75'
 {{< /code >}}
 {{< code json >}}
 {
-  "name": "ANSIBLE_HOST"
+  "max": "75"
 }
 {{< /code >}}
 {{< /language-toggle >}}
 
-secret       | 
+min          | 
 -------------|------
-description  | Name of the Sensu secret resource that defines how to retrieve the [secret][56].
-required     | true
+description  | Minimum threshold for the metric for [metric threshold evaluation][79]. You must provide a thresholds `min` value if you do not provide a `max` value.
+required     | false (if a thresholds `max` value is provided)
 type         | String
 example      | {{< language-toggle >}}
 {{< code yml >}}
-secret: sensu-ansible-host
+min: '2'
 {{< /code >}}
 {{< code json >}}
 {
-  "secret": "sensu-ansible-host"
+  "min": "2"
+}
+{{< /code >}}
+{{< /language-toggle >}}
+
+status       | 
+-------------|------
+description  | Event [check status][77] to use if the check's output metric value matches a `max` or `min` threshold in [metric threshold evaluation][79].{{% notice note %}}
```

Comment:
It is somewhat implied in min and max but would it be worthwhile to clarify what `matches` mean? Example say that a threshold matches if the actual metric value is below `min` and/or `above`.